src/overlays/actors/ovl_En_Zl2/z_en_zl2.c

```diff
@@ -15,7 +18,78 @@ void EnZl2_Destroy(Actor* thisx, GlobalContext* globalCtx);
+void func_80B50D94(EnZl2* this, GlobalContext* globalCtx);
+void func_80B50DE8(EnZl2* this, GlobalContext* globalCtx);
+void func_80B50E3C(EnZl2* this, GlobalContext* globalCtx);
+void func_80B50E90(EnZl2* this, GlobalContext* globalCtx);
+void func_80B50EE4(EnZl2* this, GlobalContext* globalCtx);
+void func_80B50F38(EnZl2* this, GlobalContext* globalCtx);
+void func_80B50F8C(EnZl2* this, GlobalContext* globalCtx);
+void func_80B50FE8(EnZl2* this, GlobalContext* globalCtx);
+void func_80B51034(EnZl2* this, GlobalContext* globalCtx);
+void func_80B51080(EnZl2* this, GlobalContext* globalCtx);
+void func_80B510CC(EnZl2* this, GlobalContext* globalCtx);
+void func_80B51118(EnZl2* this, GlobalContext* globalCtx);
+void func_80B51164(EnZl2* this, GlobalContext* globalCtx);
+void func_80B511B0(EnZl2* this, GlobalContext* globalCtx);
+void func_80B511FC(EnZl2* this, GlobalContext* globalCtx);
+void func_80B51250(EnZl2* this, GlobalContext* globalCtx);
+void func_80B512B8(EnZl2* this, GlobalContext* globalCtx);
+void func_80B51310(EnZl2* this, GlobalContext* globalCtx);
+void func_80B51A5C(EnZl2* this, GlobalContext* globalCtx);
+void func_80B51A8C(EnZl2* this, GlobalContext* globalCtx);
+void func_80B51AE4(EnZl2* this, GlobalContext* globalCtx);
+void func_80B51B44(EnZl2* this, GlobalContext* globalCtx);
+void func_80B51BA8(EnZl2* this, GlobalContext* globalCtx);
+void func_80B51C0C(EnZl2* this, GlobalContext* globalCtx);
+void func_80B51C64(EnZl2* this, GlobalContext* globalCtx);
+void func_80B51CA8(EnZl2* this, GlobalContext* globalCtx);
+void func_80B52068(EnZl2* this, GlobalContext* globalCtx);
+void func_80B52098(EnZl2* this, GlobalContext* globalCtx);
+void func_80B52108(EnZl2* this, GlobalContext* globalCtx);
+void func_80B521A0(EnZl2* this, GlobalContext* globalCtx);
+void func_80B523BC(EnZl2* this, GlobalContext* globalCtx);
+void func_80B523C8(EnZl2* this, GlobalContext* globalCtx);
+void func_80B525D4(EnZl2* this, GlobalContext* globalCtx);
+
+UNK_TYPE D_80B52810[] = {
+    0x060030C8, 0x06003C48, 0x06004048, 0x06004548, 0x06004948, 0x06004D48, 0x06005148, 0x06005D48, 0x06006148,
+};
+
+UNK_TYPE D_80B52834[] = {
+    0x06003508,
+    0x06005548,
+    0x06005948,
+};
+
+static EnZl2ActionFunc sActionFuncs[] = {
+    func_80B521A0, func_80B50BBC, func_80B50BEC, func_80B50C40, func_80B50CA8, func_80B50CFC,
+    func_80B50D50, func_80B50D94, func_80B50DE8, func_80B50E3C, func_80B50E90, func_80B50EE4,
+    func_80B50F38, func_80B50F8C, func_80B50FE8, func_80B51034, func_80B51080, func_80B510CC,
+    func_80B51118, func_80B51164, func_80B511B0, func_80B511FC, func_80B51250, func_80B512B8,
+    func_80B51310, func_80B51A5C, func_80B51A8C, func_80B51AE4, func_80B51B44, func_80B51BA8,
+    func_80B51C0C, func_80B51C64, func_80B51CA8, func_80B52068, func_80B52098, func_80B52108,
+};
+
+EnZl2PreLimbDrawFunc sOverrideLimbDrawFuncs[] = {
+    func_80B4F45C,
+};
+
+static ActorFunc sDrawFuncs[] = {
```

Comment:
this should probably be typedefed and take a zl2, its the pattern we use everywhere else atm

Reply:
oh yeah oops. will do